### DISCUSSION TITLE
no selected project

### DISCUSSION THREAD

**Discussion by fogbunny** [rfvgyhn/min-ed-launcher#142] [Q&A]
couple years ago i was playing ed. today i want to start again but an error occured named error hresult: 0x80072EE4. i couldnt solve the problem and then i found this min-ed-launcher i downloaded it i followed the steps but i cant run the game it says no selected project probably i missed something. 
![MinEdLauncher_BGxHd4poR2](https://github.com/user-attachments/assets/cc88c834-7386-4227-b7ad-96f747b389e3)
![EpicGamesLauncher_hnBJHihFM1](https://github.com/user-attachments/assets/f1552052-8044-4f04-8ee4-e57b6e78c8e2)

**Comment by rfvgyhn** [comment]
You've told it to run the live version Horizons (/edh4) but it's not installed.

For whatever reason, FDev has Epic install the legacy version by default. You'll need to tell min-ed-launcher to install and update the live version by setting `forceUpdate` in your [settings file](https://github.com/rfvgyhn/min-ed-launcher?tab=readme-ov-file#settings).

`"forceUpdate": "FORC-FDEV-DO-38-IN-40",`

**Reply by fogbunny** [reply]
Ok. tbh i don't understand these coding things. can u tell me what should i do? how do i force to install horizon? i mean i read the settings thing but im more confused rn

**Reply by fogbunny** [reply]
Ok ok i found it thank you it downloads horizons now thanks
